docs/contributors/github-workflow.md

```diff
@@ -0,0 +1,108 @@
+## GitHub workflow for El Carro
+
+This document provides an overview of the Git/GitHub workflow used by the El
+Carro project. Following the instructions in this guide helps us streamline
+development and evolution of the project.
+
+To make your first contribution to El Carro, you should follow the steps below.
+
+1.  Create a fork of the El Carro repo
+
+    -   Visit https://github.com/GoogleCloudPlatform/elcarro-oracle-operator
+    -   Click the Fork button (top right) to create a fork hosted on GitHub
+
+2.  Clone your fork to your machine Your fork should be cloned to a directory on
```

Comment:
We should recommend users use `GOMODULE111=on` so that the $GOPATH is no longer a problem. This is how our makefile already works as well.

Reply:
Great idea! I made a change to include the GO111MODULE flag.